frontend/src/lib/gql/gql-client.ts

```diff
@@ -61,6 +61,19 @@ function createGqlClient(_gqlEndpoint?: string): Client {
               if (args.input.orgId) {
                 cache.invalidate({__typename: 'OrgById', id: args.input.orgId}, 'projects');
               }
+              if (args.input.id) {
+                cache.invalidate({__typename: 'Project', id: args.input.id});
+                cache.invalidate({__typename: 'DraftProject', id: args.input.id});
+                // Note singular MyProject name for the myProjects query cache, ditto for draft
+                cache.invalidate({__typename: 'MyProject', id: args.input.id});
+                cache.invalidate({__typename: 'MyDraftProject', id: args.input.id});
+              }
+              if (result?.createProject?.createProjectResponse?.id) {
+                cache.invalidate({__typename: 'Project', id: result?.createProject?.createProjectResponse?.id});
+                cache.invalidate({__typename: 'DraftProject', id: result?.createProject?.createProjectResponse?.id});
+                cache.invalidate({__typename: 'MyProject', id: result?.createProject?.createProjectResponse?.id});
+                cache.invalidate({__typename: 'MyDraftProject', id: result?.createProject?.createProjectResponse?.id});
```

Comment:
I think only `DraftProject` is needed here.

Reply:
Removed `Project` cache invalidation in commit 27cefe5a. Keeping MyProject and MyDraftProject as mentioned in https://github.com/sillsdev/languageforge-lexbox/pull/977#discussion_r1687802315.